<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBNWelcomeToTheInterface</class>
 <widget class="QWidget" name="OBNWelcomeToTheInterface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>643</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>643</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>643</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>欢迎使用OBN软件</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/bootImage/img/tubiao.png</normaloff>:/bootImage/img/tubiao.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QFrame" name="frameWelcomeImage">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser">
          <property name="styleSheet">
           <string notr="true">/***
QTextEdit#textBrowser{
	background-color:transparent;
}***/</string>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;			&lt;img src=&quot;:/bootImage/img/logo.png&quot; /&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;		全国免费服务热线：&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:9pt;&quot;&gt;800-807-2006 &lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;国内销售热线：&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:9pt;&quot;&gt;023-65291554&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;、&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:9pt;&quot;&gt;023-89863540&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;、&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:9pt;&quot;&gt;023-65291634&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;、&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:9pt;&quot;&gt;023-89863065&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;		国际销售热线：&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:9pt;&quot;&gt;0086-023-65157761 &lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;技术服务热线： &lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:9pt;&quot;&gt;023-89863740&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;（电法仪器）&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:9pt;&quot;&gt;/ 023-89863764&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;（测井仪器）&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;		传真：&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;023-89863065&lt;/span&gt;&lt;span style=&quot; font-family:'Microsoft YaHei','Segoe UI','system-ui','Roboto','Droid Sans','Helvetica Neue','sans-serif','Tahoma','Segoe UI Symbol','Myanmar Text'; font-size:8pt; color:#171a1d;&quot;&gt;023-65291634&lt;/span&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;（海洋仪器）&lt;/span&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:8pt;&quot;&gt;/ &lt;/span&gt;&lt;span style=&quot; font-family:'Microsoft YaHei','Segoe UI','system-ui','Roboto','Droid Sans','Helvetica Neue','sans-serif','Tahoma','Segoe UI Symbol','Myanmar Text'; font-size:8pt; color:#171a1d;&quot;&gt;023-65291568&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;（地震仪器及检波器）&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:9pt;&quot;&gt;		Email&lt;/span&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;：&lt;/span&gt;&lt;a href=&quot;mailto:cdy@cgif.com.cn&quot;&gt;&lt;span style=&quot; font-family:'Liberation Serif','serif'; font-size:9pt; font-weight:600; text-decoration: underline; color:#0000ff;&quot;&gt;cdy@cgif.com.cn &lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;		地址：&lt;/span&gt;&lt;span style=&quot; font-size:10pt; color:#171a1d;&quot;&gt;重庆市高新区曾家镇兴德路&lt;/span&gt;&lt;span style=&quot; font-family:'Microsoft YaHei','Segoe UI','system-ui','Roboto','Droid Sans','Helvetica Neue','sans-serif','Tahoma','Segoe UI Symbol','Myanmar Text'; font-size:10pt; color:#171a1d;&quot;&gt;6&lt;/span&gt;&lt;span style=&quot; font-size:10pt; color:#171a1d;&quot;&gt;号&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt; color:#2e3436;&quot;&gt;		使用软件之前，为了方便后面的工作，需要先对当前的工作新建一个工区，在后面的使用过程中，&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt; color:#2e3436;&quot;&gt;		所有操作产生的数据都将存储在当前工程中: &lt;/span&gt;&lt;a href=&quot;NewProject&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#000000;&quot;&gt;新建工区&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt; color:#000000;&quot;&gt;		您也可以为您的工区进行搬家或者其他操作： &lt;/span&gt;&lt;a href=&quot;EditProject&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#000000;&quot;&gt;编辑工区&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt; color:#000000;&quot;&gt;		当您的工程已经结束或者不需要时，您可以将其进行删除：&lt;/span&gt;&lt;a href=&quot;DeleteProject&quot;&gt;&lt;span style=&quot; font-size:9pt; text-decoration: underline; color:#000000;&quot;&gt;删除工区&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QFrame" name="frameButtonGroup">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>102</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="labelProject">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>工区选择</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelLanguage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>语言选择 </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="comboBoxProject">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBoxLanguage">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>简体中文</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>English</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="buttonEnterTheSystem">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#buttonEnterTheSystem{
	background-color:&quot;#1E90FF&quot;;
}</string>
          </property>
          <property name="text">
           <string>进入系统</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
